.eslintrc.js

```diff
@@ -8,10 +9,12 @@ const config = {
       {
         anonymous: "never",
         named: "never",
-        asyncArrow: "always"
-      }
-    ]
-  }
+        asyncArrow: "always",
+      },
+    ],
+    "comma-dangle": ["error", "never"],
```

Comment:
I ended up adding these because of the change in Prettier 2. We should probably create a changelog with breaking changes related to Prettier 2.